software/control_sw/src/cosmic_fengine.py

```diff
@@ -1493,7 +1521,7 @@ def delay_tracking(self):
                                 clock_rate_hz=2048000000, invert_band = False)
                 
                 if(required_loadtime_s > (time.time_ns()*1e-9 + 1e-2)):
-                    #give a bit of room for the loadtime
+                    #give 5ms of room for the loading of the loadtime
```

Comment:
where is this 5ms? (the `if` is still `now + 10ms`)
The earlier code seems to set the boundary at 100ms, so I'd guess this can be relaxed. Though if your experience is that 10ms is sufficient then that's fine

Reply:
Mismatch between code and comment apologies. I'll go back to the 100ms I think (just in case it takes a while to set the loadtime). This is a semi arbitrary value with the interpolation loop being 1s now which is 100 times slower than we used to run it - I expect the above check to always pass.

Reply:
No problem, it's just I see you actively _changed_ the comment to be more specific :laughing: 

I think 100ms is probably fine (and seems nice and safe) but FWIW this means that the minimum allowed future load time of 100ms in the future will probably just fail here.